ansible/group_vars/all.yml

```diff
@@ -27,6 +27,11 @@ cluster_name: cluster.local
 insecure_registrys:
 #  - "gcr.io"
 
+# Required for CoreOS. CoreOS does not include a Python interpreter. The
+# pre-ansible role installs a python interpreter to /opt/bin/. For more
+# information see https://coreos.com/blog/managing-coreos-with-ansible.html
+#ansible_python_interpreter: "PATH=/opt/bin:$PATH python"
```

Comment:
Can this be automatically set in the pre-ansible role using the 'raw' module to verify things?